spec/spec.md

```diff
@@ -236,7 +244,7 @@ in the [challenge token section](#challenge-token-2) above:
 {
     "type": "https://didcomm.org/present-proof/3.0/propose-presentation",
     "id": "95e63a5f-73e1-46ac-b269-48bb22591bfa",
-    "thid": "599f3638-b563-4937-9487-dfe55099d900",
+    "pthid": "599f3638-b563-4937-9487-dfe55099d900",
```

Comment:
when you display a QR Code, you are technically forking the thread, this indicates the case where you are replying to a fork... ?

discussed on the call, considered helpful.